zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/client/CreateDocumentResult.kt

```diff
@@ -24,4 +24,24 @@ class CreateDocumentResult(
     val bestandsnaam: String,
     val bestandsomvang: Long,
     val beginRegistratie: LocalDateTime,
-)
\ No newline at end of file
+    val bestandsdelen: List<Bestandsdelen>?
+) {
+    fun getBestandsdelenIdFromUrl(): String {
```

Comment:
Where is this used? Only in tests? 


Is this the id of the bestandsdeel? If so, it's strange to always pick the first one from an array.

Reply:
I think this is a leftover from an old attempt. I will clean it up